.github/workflows/test-docs-build.yml

```
@@ -0,0 +1,21 @@
+name: Test Docs Build
+
+on:
+  pull_request:
+    branches:
+      - support/v0.5
```

Ideally we would want this to run on all PRs (will be merged back to `dev` from the `support/v0.5` branch). Anything else before this can be merged, @eike-hass ? I haven't tested anything.

```suggestion
on:
  push:
    branches:
      - main
      - dev
  pull_request:
    branches:
      - main
      - dev
      - epic/*
      - support/*
```

---

Maybe I haven't thought this through 100% either :trollface:

---

That makes sense. Does the wiki automatically pull in those branches or is it pinned to `support/v0.5` on your side for now? 

I was just concerned about PRs breaking the wiki over time and only noticing during/after a release.

---

No. As soon as you merge this PR, I will add the `support/v0.5` branch as a git submodule to the wiki.

Eventually the new cli will have an easy version managing tool which allows you to add/remove versions in the wiki repo. We will also use gitpod as a tool there so that you won't need to clone the wiki repo and testing and making those changes is as less overhead for you as possible.

---

I mean we only pull from a specific set of branches, right? So if those branches are checked for braking links we will never have broken links in the wiki.
The problem is mainly if you are creating bigger feature branches that you won't to merge after months of coding on it and then you realise basically your whole branch contains broken links and you need to fix them before you merge into a release branch or create a new release branch :thinking:

---

Makes sense 👍

---

The workflow will only run on a branch if it is present on it. So we would merge the workflow together with the build system. I think @cycraig initial thought and suggestion is correct. Will commit the suggestion.